src/DSP.jl

```diff
@@ -7,43 +7,20 @@ module DSP
 macro importffts()
     quote
         using AbstractFFTs
-        if VERSION >= v"0.7.0-DEV.602"
-            using FFTW
-        end
+        using FFTW
     end
 end
 
 @importffts
+using Compat.LinearAlgebra: mul!, rmul!
```

Comment:
Can leave out the `Compat`

Reply:
Can things marked using/import Compat: ... be removed in general, or only Compat.LinearAlgebra to LinearAlgebra

Reply:
We shouldn’t need Compat at all unless We are using 1.1 features not available in 1.0.